exporter/dynatraceexporter/README.md

```diff
@@ -261,6 +263,45 @@ User should calculate this as `num_seconds * requests_per_second` where:
 
 Default: `5000`
 
+### resource_to_telemetry_conversion (Optional)
+
+When `resource_to_telemetry_conversion.enabled` is set to `true`, all resource
+attributes will be included as metric attributes.
```

Comment:
Can you specify which of the two takes precedence in case the attribute names would clash?

Reply:
I noticed that this is not documented in the `resourcetotelemetry` helper's readme, so we should actually not mention the current precedence here, as it would be unedfined behavior that can change at any time.

Reply:
yeah i'll remove it